docs/quickstart/Azure.md

```diff
@@ -0,0 +1,126 @@
+## Requirements 
+To get started running Coder on Azure, you will need an Azure account with the capability to create new Virtual Machines. 
+
+## Create An Azure VM
+
+From the Azure Portal, navigate to the Virtual Machines Dashboard. Click Create, and select creating a new Azure Virtual machine .
+
+<img src="../images/quickstart/azure/azure1.jpg">
+
+
+This will bring you to the `Create a virtual machine page`. Select the subscription group of your choice, or create one if necessary.
```

Comment:
```suggestion
This will bring you to the `Create a virtual machine` page. Select the subscription group of your choice, or create one if necessary.
```

Reply:
assuming this was a typo